homeassistant/components/transmission/translations/en.json

```diff
@@ -23,6 +23,7 @@
                     "name": "Name",
                     "password": "Password",
                     "port": "Port",
+                    "ssl": "Uses an SSL certificate",
```

Comment:
This wording is weird, maybe just `Use SSL`

Here is the octoprint integration for example
![image](https://user-images.githubusercontent.com/2403652/220766964-dd966b17-3bb9-4983-8071-54d113c859e7.png)